util-taglib/src/com/liferay/taglib/ui/InputEditorTag.java

```diff
@@ -282,7 +283,15 @@ protected String getPage() {
 	}
 
 	protected String getPage(String editorName) {
-		return "/html/js/editor/" + editorName + ".jsp";
+		if (_page == null) {
+			ThemeDisplay themeDisplay = (ThemeDisplay)request.getAttribute(
+				WebKeys.THEME_DISPLAY);
+
+			_page = themeDisplay.getPathJavaScript() + "/editor/" +
+				editorName + ".jsp";
+		}
```

Comment:
This if block is a bug, because once set, it will never again consider the editorName, and will return the same value for all editors.